sdk/search/Azure.Search.Documents/src/Models/SearchModelFactory.cs

```diff
@@ -211,7 +211,26 @@ public static partial class SearchModelFactory
             SearchResourceCounter dataSourceCounter,
             SearchResourceCounter storageSizeCounter,
             SearchResourceCounter synonymMapCounter) =>
-            new SearchServiceCounters(documentCounter, indexCounter, indexerCounter, dataSourceCounter, storageSizeCounter, synonymMapCounter);
+                SearchServiceCounters(documentCounter, indexCounter, indexerCounter, dataSourceCounter, storageSizeCounter, synonymMapCounter, skillsetCounter: null);
```

Comment:
We should [EditorBrowsable(Never)] the old one so customers only see the up to date version.

Reply:
Also, why not initialize `skillsetCounter` to an empty `SearchResourceCounter`?

Reply:
I am making the first change. I am not sure what is a good default for `usage` to initialize a `SearchResourceCounter`.